wagl/standardise.py

```diff
@@ -285,31 +287,46 @@ def card4l(level1, granule, workflow, vertices, method, pixel_quality, landsea,
         lon_lat_grp = res_group[GroupName.LON_LAT_GROUP.value]
 
         # TODO: supported acqs in different groups pointing to different response funcs
-        # tp5 files
-        tp5_data, _ = format_tp5(acqs, ancillary_group, sat_sol_grp,
+        json_data, _ = format_json(acqs, ancillary_group, sat_sol_grp,
                                  lon_lat_grp, workflow, root)
 
         # atmospheric inputs group
         inputs_grp = root[GroupName.ATMOSPHERIC_INPUTS_GRP.value]
 
         # radiative transfer for each point and albedo
-        for key in tp5_data:
+        for key in json_data:
             point, albedo = key
 
             log.info('Radiative-Transfer', point=point, albedo=albedo.value)
+
             with tempfile.TemporaryDirectory() as tmpdir:
 
                 prepare_modtran(acqs, point, [albedo], tmpdir, modtran_exe)
 
-                # tp5 data
-                fname = pjoin(tmpdir,
-                              tp5_fmt.format(p=point, a=albedo.value))
-                with open(fname, 'w') as src:
-                    src.writelines(tp5_data[key])
+                point_dir = pjoin(tmpdir, POINT_FMT.format(p=point))
+                workdir = pjoin(point_dir, ALBEDO_FMT.format(a=albedo.value))
+                # json data
+                json_mod_infile = pjoin(tmpdir, json_fmt.format(p=point, a=albedo.value))
+
+                with open(json_mod_infile, 'w') as src:
+
+                    json_string = json_data[key]
+
+                    if albedo == Albedos.ALBEDO_TH:
+
+                        json_string["MODTRAN"][0]["MODTRANINPUT"]["SPECTRAL"]["FILTNM"] = "%s/%s" % (workdir, json_string["MODTRAN"][0]["MODTRANINPUT"]["SPECTRAL"]["FILTNM"])
+                        json_string["MODTRAN"][1]["MODTRANINPUT"]["SPECTRAL"]["FILTNM"] = "%s/%s" % (workdir, json_string["MODTRAN"][1]["MODTRANINPUT"]["SPECTRAL"]["FILTNM"])
+
+                    else:
+
+                        json_string["MODTRAN"][0]["MODTRANINPUT"]["SPECTRAL"]["FILTNM"] = "%s/%s" % (workdir, json_string["MODTRAN"][0]["MODTRANINPUT"]["SPECTRAL"]["FILTNM"])
+
+                    d = json.dumps(json_string, cls=JsonEncoder, indent=4)
+
+                    src.writelines(d)
+
+                run_modtran(acqs, inputs_grp, workflow, nvertices, point,[albedo], modtran_exe,tmpdir, root, compression,filter_opts)
 
-                run_modtran(acqs, inputs_grp, workflow, nvertices, point,
```

Comment:
The reasoning behind the multiline call to functions, is to avoid having lines that are too long. PEP8 recommends no longer than 79 characters per line.

Reply:
check